config/routes.rb

```diff
@@ -1,6 +1,5 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index, :show]
+  end
 end
```

Comment:
- [ ] According to the wireframe the `users/index` view should render on the root route. Please add this code to the `routes.rb` file.
  - `root 'users#index'`